Codeinwp/otter-blocks
# Use Optimole image links for Patterns

Closes #1608 .


### Summary


Replace Patterns image links with Optimole links.

### Screenshots 

![image](https://user-images.githubusercontent.com/17597852/235939609-d1580040-8bc4-447f-b54e-0f3e428104a3.png)


----

### Test instructions


- Make sure that image links are not broken.
- Patterns look the same (no dimension change).



---- 

### Checklist before the final review
